notion/README.md

```diff
@@ -0,0 +1,39 @@
+# Agent Check: Notion
+
+## Overview
+
+Notion is a Connected Workspace. It's where modern teams create and share docs, take notes, manage projects, and organize knowledge, all in one place. Install Notion's integration with Datadog to manage and monitor your workspace activity in [Datadog Cloud SIEM][1]. You can import your workspace's audit logs for real-time monitoring, alerting, and analysis. From there, you can detect and investigate potential security issues, suspicious behavior, and troubleshoot access with confidence and ease.
+
+## Setup
+
+Datadog setup
+Open Datadog and go to Integrations > Integrations.
+
+Search for "Notion".
```

Comment:
This shows on the Notion tile so the first two steps are not needed since they will already be on the Notion tile

Reply:
@emarsha94 I've updated this in the latest commit. Given that I made this change last week, I'd kindly ask that Datadog review changes as they are requested as that will limit the back and forth and expedite this process.